### PR TITLE
[Tests-Only] Split Resharing test suites

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
split some of the `webUIResharing` tests into a smaller suite 
- webUIResharing1
- webUIResharing2

## Related Issue
https://github.com/owncloud/web/issues/4876

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by swoichha — COMMENTED** [review]
LGTM

File: .drone.star [hunk 1]
```diff
@@ -331,8 +336,14 @@ config = {
...
 				'webUISharingPermissionToRoot': 'OCISSharingPermissionToRoot',
-				'webUIResharing': 'OCISResharing', # for now run this suite by itself see https://github.com/owncloud/ocis/issues/736
-				'webUIResharingToRoot': 'OCISResharingToRoot',
+				# for now run this suite by itself see https://github.com/owncloud/ocis/issues/736
+				'OCISResharing1': [
+					'webUIResharing1'
+				],
+				'OCISResharing2': [
+					'webUIResharing2',
+					'webUIResharingToRoot'
```
Comment (dpakach) [inline .drone.star:345]: I dont think we need to run `ToRoot` suites in ocis

Comment (saw-jan) [reply .drone.star:345]: I was also thinking the same but it was already there, so I just included it.

File: .drone.star [hunk 2]
```diff
@@ -62,7 +62,10 @@ config = {
 					'webUIFilesSearch'
 				],
 				'webUIMoveFilesFolders': 'Move',
-				'webUIResharing': 'Resharing',
+				'Resharing': [
```
Comment (kiranparajuli589) [inline .drone.star:65]:
```suggestion
				'webUIResharing': [
```

**Review by individual-it — APPROVED** [review]

**Pull request merged**